mods/default/mapgen.lua

```diff
@@ -312,6 +312,32 @@ function default.register_mgv6_ores()
 		y_min          = -31000,
 	})
 
+	-- Silver
+
+	minetest.register_ore({
+		ore_type       = "scatter",
+		ore            = "default:stone_with_silver",
+		wherein        = "default:stone",
+		clust_scarcity = 11*11*11,
+		clust_num_ores = 4,
+		clust_size     = 11,
```

Comment:
These cluster parameters are unnecessary and intensive. A 11x11x11 cluster with 4 ores with scarcity 11*11*11 is just 4 ores per 11x11x11 volume so no different to ...
```
		clust_scarcity = 11*11*11 / 4,
		clust_num_ores = 1,
		clust_size     = 1,
```
... but far more intensive to generate.

Reply:
By `11*11*11 / 4` i meant 1331 / 4 = 332.75 = roughly 7*7*7.
And you missed out `clust_size     = 1,`